swerve_controller/test/config/swervebot_plotjuggler.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root version="2.3.8">
+ <tabbed_widget name="Main Window" parent="main_window">
+  <plotmatrix rows="2" columns="2" tab_name="steering positions">
+   <plot row="0" col="0">
+    <range left="0.007000" bottom="0.000009" top="0.000009" right="1.056000"/>
+    <limitY/>
+    <curve G="189" custom_transform="noTransform" B="34" name="/swervebot/joint_states/leg_lf_joint/pos" R="188"/>
+    <transform value="noTransform"/>
+   </plot>
+   <plot row="1" col="0">
+    <range left="0.007000" bottom="-0.000794" top="-0.000793" right="1.056000"/>
+    <limitY/>
+    <curve G="201" custom_transform="noTransform" B="56" name="/swervebot/joint_states/leg_lh_joint/pos" R="26"/>
+    <transform value="noTransform"/>
+   </plot>
+   <plot row="0" col="1">
+    <range left="0.007000" bottom="-0.000007" top="0.000006" right="1.056000"/>
+    <limitY/>
+    <curve G="119" custom_transform="noTransform" B="180" name="/swervebot/joint_states/leg_rf_joint/pos" R="31"/>
```

Comment:
sure you want to submit this?

Reply:
It is indeed mainly for me. I will remove it from the pull request